app/client/cli/actor.go

```diff
@@ -0,0 +1,217 @@
+			// TODO (team): passphrase is currently not used since there's no keybase yet, the prompt is here to mimick the real world UX
+			_ = Credentials(pwd)
+
+			msg := &types.MessageStake{
+				PublicKey:     pk.PublicKey().Bytes(),
+				Chains:        chains,
+				Amount:        amount,
+				ServiceUrl:    serviceURI,
+				OutputAddress: []byte{},     // TODO(deblasis): 👀
+				Signer:        pk.Address(), // TODO(deblasis): figure out where I should get this from (https://github.com/pokt-network/pocket/pull/169#discussion_r953186602)
+				ActorType:     cmdDef.ActorType,
+			}
+
+			codec := sharedTypes.GetCodec()
+			anyMsg, err := codec.ToAny(msg)
+
+			signature, _ := pk.Sign(msg.GetSignBytes())
+
+			tx := &utilityTypes.Transaction{
+				Msg: anyMsg,
+				Signature: &utilityTypes.Signature{
+					Signature: signature,
+					PublicKey: pk.PublicKey().Bytes(),
+				},
+				Nonce: "", // TODO(deblasis): figure out where I should get this from
+			}
+
+			j, err := json.Marshal(tx)
+			if err != nil {
+				return err
+			}
+
+			// TODO(deblasis): we need a single source of truth for routes, the empty string should be replaced with something like a constant that can be used to point to a specific route
+			// perhaps the routes could be centralized into a map[string]Route in #176 and accessed here
+			// I will do this in #169 since it has commits from #176 and #177
+			resp, err := QueryRPC("", j)
+			if err != nil {
+				return err
+			}
+			fmt.Println(resp)
+
+			return nil
+		},
+	}
+	cmds = append(cmds, stakeCmd)
+
+	editStakeCmd := &cobra.Command{
+		Use:   "EditStake <fromAddr> <amount>",
+		Short: "EditStake <fromAddr> <amount>",
+		Long:  fmt.Sprintf(`Stakes a new <amount> for the %s actor with address <fromAddr>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(pocket/issues/150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			amount := args[1]
+
+			_ = &types.MessageEditStake{
+				Address:    pk.Address(),
+				Chains:     []string{}, // TODO(deblasis): 👀
+				Amount:     amount,
+				ServiceUrl: "",       // TODO(deblasis): 👀
+				Signer:     []byte{}, // TODO(deblasis): 👀
+				ActorType:  cmdDef.ActorType,
+			}
+
+			return nil
+		},
+	}
+	cmds = append(cmds, editStakeCmd)
+
+	unstakeCmd := &cobra.Command{
+		Use:   "Unstake <fromAddr>",
+		Short: "Unstake <fromAddr>",
+		Long:  fmt.Sprintf(`Unstakes the prevously staked tokens for the %s actor with address <fromAddr>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(pocket/issues/150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			_ = &types.MessageUnstake{
+				Address:   pk.Address(),
+				Signer:    []byte{}, // TODO(deblasis): 👀
```

Comment:
'Signer' may be different than 'Address' which is the operator.

Signer must either be the 'Output Address' or the 'Operator Address'

Reply:
I see, should we have two separate subcommands? One for `custodial` and one for `non-custodial` like V0?

Reply:
`two separate subcommands` - Can you create a followup issue to add support for `non-cusotidal CLI support` to avoid increasing the scope of this PR.